lib/Jejik/MT940/Parser/AbstractParser.php

```diff
@@ -159,57 +172,63 @@ protected function splitTransactions($text)
      * Parse a statement chunk
      *
      * @param string $text Statement text
+     *
      * @return \Jejik\MT940\Statement
-     * @throws \RuntimeException if the chunk cannot be parsed
      */
-    protected function statement($text)
+    protected function statement($text): ?\Jejik\MT940\Statement
     {
         $text = trim($text);
         if (($pos = strpos($text, ':20:')) === false) {
             throw new \RuntimeException('Not an MT940 statement');
         }
 
         $this->statementHeader(substr($text, 0, $pos));
+
         return $this->statementBody(substr($text, $pos));
     }
 
     /**
      * Parse a statement header
      *
      * @param string $text Statement header text
+     *
      * @return void
      */
-    protected function statementHeader($text)
+    protected function statementHeader($text): void
     {
     }
 
     /**
      * Parse a statement body
      *
      * @param string $text Statement body text
-     * @return \Jejik\MT940\Statement
+     *
+     * @return \Jejik\MT940\Statement|null
+     * @throws \Exception
      */
-    protected function statementBody($text)
+    protected function statementBody($text): ?\Jejik\MT940\Statement
```

Comment:
Don't use FQCN, add a `use` statement instead.

Reply:
Changed from FQCN to use.